src/pages/Detail/Aside/CheckBox.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CheckBox = ({ toggleCheckbox }) => {
+  return (
+    <CheckBoxSection>
+      <div>
+        <input type="checkbox" onChange={toggleCheckbox} />
+        <label>단독홀</label>
+      </div>
+      <Price fontSize="17px">
+        ₩6000
+        <Text>/ 시간</Text>
+      </Price>
+    </CheckBoxSection>
+  );
+};
+
+export default CheckBox;
+
+const CheckBoxSection = styled.div`
+  ${props => props.theme.flexMixIn('space-between', 'center')}
```

Comment:
구조분해 할당 할  수 있습니다!
다른 부분도 적용시켜주세요:)
```suggestion
  ${({theme}) => theme.flexMixIn('space-between', 'center')}
```